### ISSUE TITLE
Cannot read property ... of undefined

### ISSUE THREAD

**Issue by i3laze** [DeutscheMark/homebridge-kodi#9]
Happy New Year!) 

I’ve skimmed new readme and then blindly upgraded 0.2.2 to 0.3.0.
Then the plugin couldn’t start without defining these new variables in config:
controls, channels, volume, scan ...

Looks like your optional variables aren’t optional yet at all ;)

As a workaround had to define them manually:
```
"television": {
                "controls": {},
                "tv": {}
            }
```
P.S. Can’t remove Controls from HomeKit. That one totally isn’t optional so far.

P.P.S. Remind me how to remove a progress lightbulb accessory, as “turn off the lights” invokes a pause.

**Comment by DeutscheMark** [comment]
Happy New Year! :)
Thanks for pointing that one out.
I will fix the problem asap with v0.3.1.

**Comment by DeutscheMark** [comment]
Fixed in v0.3.1.
